build/media-suite_compile.sh

```diff
@@ -2432,27 +2438,107 @@ if [[ $mpv != n ]] && pc_exists libavcodec libavformat libswscale libavfilter; t
+            # Audio option time! This goes in twos; we're unconditionally disabling non-Windows options and warning the user.
+            elif [[ $option =~ ^--enable-(alsa|audiounit|coreaudio|oss-audio|pipewire|pulse|sndio) ]]; then
+                meson_opts+=("${option/--enable-/-D}=disabled")
+                do_simple_print "${orange}Disabling option '${option}' as it's not usable on Windows"'!'"${reset}"
+            elif [[ $option =~ ^--enable-(jack|openal|opensles|sdl2-audio|wasapi) ]]; then
+                meson_opts+=("${option/--enable-/-D}=enabled")
+            elif [[ $option =~ ^--disable-(jack|openal|opensles|sdl2-audio|wasapi) ]]; then
+                meson_opts+=("${option/--disable-/-D}=disabled")
+            # Video corner, same as with audio
+            elif [[ $option =~ ^--enable-(cocoa|drm|egl-android|*-drm|*-wayland|*-x11|*-cocoa|gbm|rpi|vdpau|vdpau-*|vaapi|vaapi-*|wayland|x11|xv) ]]; then
+                meson_opts+=("${option/--enable-/-D}=disabled")
+                do_simple_print "${orange}Disabling option '${option}' as it's not usable on Windows"'!'"${reset}"
+            elif [[ $option =~ ^--enable-(caca|d3d11|direct3d|egl|egl-*|plain-gl|gl|gl-*|jpeg|libplacebo|sdl2-video|shaderc|sixel|spirv-cross|vulkan) ]]; then
+                meson_opts+=("${option/--enable-/-D}=enabled")
+            elif [[ $option =~ ^--disable-(caca|d3d11|direct3d|egl|egl-*|plain-gl|gl|gl-*|jpeg|libplacebo|sdl2-video|shaderc|sixel|spirv-cross|vulkan) ]]; then
+                meson_opts+=("${option/--disable-/-D}=disabled")
+            # hwaccel features
+            elif [[ $option =~ ^--enable-(android-media-ndk|ios-gl|rpi-mmal|videotoolbox-gl) ]]; then
+                meson_opts+=("${option/--enable-/-D}=disabled")
+                do_simple_print "${orange}Disabling option '${option}' as it's not usable on Windows"'!'"${reset}"
+            elif [[ $option =~ ^--enable-(cuda-hwaccel|cuda-interop|d3d-hwaccel|d3d9-hwaccel|gl-dxinterop-d3d9) ]]; then
+                meson_opts+=("${option/--enable-/-D}=enabled")
+            elif [[ $option =~ ^--disable-(cuda-hwaccel|cuda-interop|d3d-hwaccel|d3d9-hwaccel|gl-dxinterop-d3d9) ]]; then
+                meson_opts+=("${option/--disable-/-D}=disabled")
+            # Mac-only options, we might as well catch and squash them
+            elif [[ $option =~ ^--enable-(macos-*|swift-*) ]]; then
+                meson_opts+=("${option/--enable-/-D}=disabled")
+                do_simple_print "${orange}Disabling option '${option}' as it's not usable on Windows"'!'"${reset}"
+            # Manpages corner
+            elif [[ $option =~ ^--enable-(html-build|manpage-build) ]]; then
+                meson_opts+=("${option/--enable-/-D}=enabled")
+            elif [[ $option =~ ^--disable-(html-build|manpage-build) ]]; then
+                meson_opts+=("${option/--disable-/-D}=disabled")
+            elif [[ $option =~ ^--enable-pdf-build ]]; then
```

Comment:
Remnant of the old code, which silently and unconditionally disabled PDF doc builds, but this needs to be tested